<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>音频播放器</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/images/images/icon.png</normaloff>:/images/images/icon.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(50,50,50);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QmyVideoWidget" name="videoWidget" native="true">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTipDuration">
          <number>0</number>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(60,60,60);

</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>9</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QPushButton" name="btnAdd">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/images/images/open.png</normaloff>:/images/images/open.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnPause">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/images/images/622.png</normaloff>:/images/images/622.png</iconset>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPlay">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/images/images/620.png</normaloff>:/images/images/620.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnStop">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/images/images/624.png</normaloff>:/images/images/624.png</iconset>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSound">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/images/images/volumn.png</normaloff>:/images/images/volumn.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sliderVolumn">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QSlider::handle:horizontal{
width:8px;
background-color:rgb(255,255,255);
margin:-8px 0px -8px 0px;
border-radius:1px;
}
QSlider::groove:horizontal{
height:2px;
background-color:rgb(219,219,219);
}
QSlider::add-page:horizontal{
background-color:rgb(219,219,219);
}
QSlider::sub-page:horizontal{
background-color:rgb(26,217,110);
}
 </string>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="font">
          <font>
           <family>微软雅黑 Light</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255,255,255);</string>
         </property>
         <property name="text">
          <string>全屏</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/images/images/full.png</normaloff>:/images/images/full.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="windowOpacity">
       <double>1.000000000000000</double>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>9</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QLabel" name="LabCurMedia">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑 Light</family>
           <stylestrategy>PreferDefault</stylestrategy>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255,255,255);</string>
         </property>
         <property name="text">
          <string>无曲目</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sliderPosition">
         <property name="styleSheet">
          <string notr="true">QSlider::handle:horizontal{
width:12px;
background-color:rgb(7,103,242);
margin:-5px 0px -5px 0px;
border-radius:6px;
}
QSlider::groove:horizontal{
height:2px;
background-color:rgb(219,219,219);
}
QSlider::add-page:horizontal{background-color:rgb(219,219,219);}
QSlider::sub-page:horizontal{background-color:rgb(26,217,110);}
 </string>
         </property>
         <property name="tracking">
          <bool>false</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::NoTicks</enum>
         </property>
         <property name="tickInterval">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="LabRatio">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑 Light</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255,255,255);</string>
         </property>
         <property name="text">
          <string>进度</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>828</width>
     <height>23</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255,255,255);</string>
   </property>
   <widget class="QMenu" name="menu">
    <property name="tabletTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255,255,255);</string>
    </property>
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="action_open"/>
    <addaction name="action_close"/>
    <addaction name="action_exit"/>
    <addaction name="action_grab"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action_open">
   <property name="text">
    <string>打开文件</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
    </font>
   </property>
  </action>
  <action name="action_close">
   <property name="text">
    <string>关闭文件</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
    </font>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
    </font>
   </property>
  </action>
  <action name="action0_5x">
   <property name="text">
    <string>0.5x</string>
   </property>
  </action>
  <action name="action0_75x">
   <property name="text">
    <string>0.75x</string>
   </property>
  </action>
  <action name="action1_0x">
   <property name="text">
    <string>1.0x</string>
   </property>
  </action>
  <action name="action1_25x">
   <property name="text">
    <string>1.25x</string>
   </property>
  </action>
  <action name="action1_5x">
   <property name="text">
    <string>1.5x</string>
   </property>
  </action>
  <action name="action2_0x">
   <property name="text">
    <string>2.0x</string>
   </property>
  </action>
  <action name="action_grab">
   <property name="text">
    <string>截图</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
    </font>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QmyVideoWidget</class>
   <extends>QWidget</extends>
   <header location="global">qmyvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
